src/main/resources/biz_de.properties

```diff
@@ -680,6 +682,18 @@ MongoUserAccountExportJobFactory.description = Exportiert Anwender in eine CSV o
 MongoUserAccountExportJobFactory.label = Anwender exportieren
 MongoUserAccountImportJobFactory.description = Importiert Anwender aus einer CSV oder Excel Datei.
 MongoUserAccountImportJobFactory.label = Anwender importieren
+Month.april = April
+Month.august = August
+Month.december = Dezember
+Month.february = Februar
+Month.january = Januar
+Month.july = Juli
+Month.june = Juni
+Month.march = März
+Month.may = Mai
+Month.november = November
+Month.october = Oktober
+Month.september = September
```

Comment:
Months are already defined in sirius-kernel:
https://github.com/scireum/sirius-kernel/blob/4629d550c6d9c6ff79d15ba6befdef63a456dd9b/src/main/resources/kernel_it.properties#L29
https://github.com/scireum/sirius-kernel/blob/4629d550c6d9c6ff79d15ba6befdef63a456dd9b/src/main/java/sirius/kernel/nls/NLS.java#L588